README.md

```diff
@@ -29,6 +29,22 @@ See [mbed_lib.json](mbed_lib.json) for all configurable options.
 
 See [test README](TESTS/mqtt/README.md) to find out about tests-specific configuration configuration.
 
+### Building with CLI V2
+
+Assuming you have this repository cloned or somehow available at the `<your-project-root>/mbed-mqtt` directory.
+
+Add to your project's `CMakeLists.txt` file the following lines:
+
+```
+add_subdirectory(mbed-mqtt)
+
+target_link_libraries(${APP_TARGET} mbed-mqtt)
+target_link_libraries(${APP_TARGET} mbed-mbedtls)
+target_link_libraries(${APP_TARGET} mbed-nanostack)
```

Comment:
why nanostack is required , shouldn't this be up to a consumer of this library (mqtt) to choose a stack (netsockets/nanostack,  etc) ?

Reply:
Well, it *is* up to a consumer here, since it is just a suggestion in `README.md` for a user. It provides an example of how you link all dependencies while linking against `mbed-mqtt` library.

It now fails to compile, though, when I began tinkering with `STATIC` declaration you've mentioned above: `TCPSocket.h: No such file or directory`. I need some time to figure out how properly declare dependency on a network stack.

Anyway, thank you very much for the review so far!

Reply:
It now have `mbed-netsocket` explicitly declared as it's dependency.